### PR TITLE
Update webpack version, add setup instructions

### PR DESCRIPTION
**Pull request by ajpal**
Fixes an issue with webpack that I ran into while trying to get a local setup of this repo, and also adds some setup instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,21 @@
 [![codecov](https://codecov.io/gh/code-dot-org/maze/branch/master/graph/badge.svg)](https://codecov.io/gh/code-dot-org/maze)
 
 Standalone repo for the Maze app type
+### Setup Your Project
+
+Check this project out from source:
+
+    git clone git@github.com:code-dot-org/craft.git
```
Comment (Hamms) [inline README.md]:
```suggestion
    git clone git@github.com:code-dot-org/maze.git
```

File: README.md [hunk 2]
```diff
@@ -5,3 +5,21 @@
...
+    cd craft
```
Comment (Hamms) [inline README.md]:
```suggestion
    cd maze
```

File: README.md [hunk 3]
```diff
@@ -5,3 +5,21 @@
...
+
+    yarn install
+    
+Now you should be able to run all the tests:
+
+    yarn test
+    
+And spin up a development build of your new project:
+
+    npm run build
```
Comment (Hamms) [inline README.md]:
```suggestion
    yarn build
```

File: README.md (hunk 1 above)
Comment (ajpal) [reply README.md]: Whoops, thanks for catching!

**Pull request merged**